### ISSUE TITLE
Fix file browser and node canvas ui scale

### ISSUE THREAD

**Issue by MiloHX** [armory3d/armorpaint#2]
Give user option to scale the UI up for HiDPI screen. 
I am using a 4K monitor on Ubuntu and I didn't find a way to scale the UI up. I can switch back to Windows 10 and scale the whole thing there, but still, proper HiDPI support will be much appreciated.

/Milo

**Comment by luboslenco** [comment]
Still a bit awkward but added an option to scale UI at http://armorpaint.org/manual/#toc7.1.

**Comment by MLHX** [comment]
Great! I tested and it was already usable! There are something not scaling correctly (x2.5):
Project/Assets Tabs:
![screenshot from 2018-02-28 08-25-35](https://user-images.githubusercontent.com/17811339/36762985-0797f6f8-1c61-11e8-942a-9f93583e35ad.png)
Drop down lists:
![screenshot from 2018-02-28 08-27-57](https://user-images.githubusercontent.com/17811339/36763048-504883fe-1c61-11e8-8993-5cf02a5e4775.png)
Import popup window:
![screenshot from 2018-02-28 08-29-59](https://user-images.githubusercontent.com/17811339/36763141-a59d3746-1c61-11e8-9da5-91ef4bb67715.png)

Hopefully once the UI scaling feature is robust, it will be implemented into the UI Canvas editor of Armory too, since I encountered the same problem there.
